docs/modules/druid/pages/usage-guide/configuration-and-environment-overrides.adoc

```diff
@@ -6,7 +6,14 @@ IMPORTANT: Overriding certain properties which are set by the operator (such as
 
 == Configuration Properties
 
-For a role or role group, at the same level of `config`, you can specify: `configOverrides` for the `runtime.properties`. For example, if you want to set the `druid.server.http.numThreads` for the router to 100 adapt the `routers` section of the cluster resource like so:
+For a role or role group, at the same level of `config`, you can specify: `configOverrides` for the following files:
+
+- `runtime.properties`
+- `jvm.config`
+- `log4j2.properties`
```

Comment:
I think we should not list this here, as we don't do it for other operators and there is a custom logging section
```suggestion
```